src/odemis/driver/technolution.py

```diff
@@ -0,0 +1,1323 @@
+    camera (mppc sensor) for acquiring the image data. It is also connected to the Scan and Acquisition module (SAM),
+    which triggers the scanner on the SEM to move the electron beam. Moreover it controls the de-scanner which counter
+    scans the scanner movement to ensure that the collected signal always hits the center of each mppc cell on the
+    detector.
+    """
+
+    def __init__(self, name, role, host, children={}, externalStorage={}, **kwargs):
+        """
+        Initialize the Acquisition server and the connection with the ASM API.
+
+        :param name (str): Name of the component
+        :param role (str): Role of the component
+        :param host (str): URL of the host (ASM)
+        :param children (dict): dictionary containing HW components and there respective configuration
+        :param externalStorage (dict): dictionary holding keys with the username, password, host and directory of the
+        external storage
+        :param kwargs:
+        """
+
+        super(AcquisitionServer, self).__init__(name, role, **kwargs)
+
+        self._host = host
+        # Use session object avoids creating a new connection for each message sent
+        # (note: Session() auto-reconnects if the connection is broken for a new call)
+        self._session = Session()
+
+        # Try (3 times) the connection with the host and stop any acquisition if already one was in progress
+        for i in range(1, 4):
+            try:
+                self.asmApiPostCall("/scan/finish_mega_field", 204)
+                break
+            except Exception as error:
+                logging.warning("Try number %s of establishing a connecting with the ASM host failed.\n"
+                                "Received the error: \n\t %s" % (i, error))
+        else:
+            logging.error("Could not connect with the ASM host.\n"
+                          "Check if the connection with the host is available and if the host URL is entered "
+                          "correctly.")
+            raise HwError("Could not connect with the ASM host.\n"
+                          "Check if the connection with the host is available and if the host URL is entered "
+                          "correctly.")
+
+        # NOTE: Do not write real username/password here since this is published on github in plaintext!
+        # example = ftp://username:password@example.com/Pictures
+        self.externalStorageURL = model.StringVA('ftp://%s:%s@%s.com/%s' %
+                                                 (externalStorage["username"],
+                                                  externalStorage["password"],
+                                                  externalStorage["host"],
+                                                  externalStorage["directory"]),
+                                                 setter=self._setURL)
+        # VA's for calibration loop
```

Comment:
-> VA to switch between calibration and acquisition mode (megafield acquisition)